website/docs/components/accordion/partials/code/how-to-use.md

```diff
@@ -147,12 +147,12 @@ The `@forceState` argument enables you to implement expand/collapse all function
   <div class="doc-accordion-flex-layout">
     <Hds::Text::Display @size="300">Examination period</Hds::Text::Display>
     <Hds::Button
-      @text={{if (eq this.state "open") "Collapse all" "Expand all"}}
-      @icon={{if (eq this.state "open") "unfold-close" "unfold-open"}}
-      @color="tertiary" @size="small" {{on "click" this.toggleState}}
+      @text={{if (eq this.accordionState "open") "Collapse all" "Expand all"}}
```

Comment:
[issue?] by re-using the `this.accordionState` variable _I think_ the two examples become entangled (one impacts on the other); maybe better to keep them independent?